Journal/Journal/View Cell/EntryTableViewCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  EntryTableViewCell.swift
+//  Journal
+//
+//  Created by Norlan Tibanear on 8/9/20.
+//  Copyright © 2020 Norlan Tibanear. All rights reserved.
+//
+
+import UIKit
+
+class EntryTableViewCell: UITableViewCell {
+    
+    // Outlets
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var dateLabel: UILabel!
+    @IBOutlet var entryLabel: UILabel!
+    
+    static let reuseIdentifier = "JournalCell"
+    
+    // Properties
+    
+    var entry: Entry? {
+        didSet {
+            updateViews()
+        }
+    }
+    
+    private func updateViews() {
+        guard let entry = entry else { return }
+        
+        let formatter3 = DateFormatter()
+        formatter3.dateFormat = "HH:mm E, d MMM y"
+        
+        titleLabel.text = entry.title
+        dateLabel.text = entry.timestamp?.description
```

Comment:
Making date formatters each time they need to be used is a little inefficient. Also, it doesn't appear that you even use the formatter here for some reason? Here is an alternative you can put in a helper file:

``` swift
extension Date {
    var shortString: String {
        return DateFormatter.shortFormatter.string(from: self)
    }
}

extension DateFormatter {
    static let shortFormatter: DateFormatter = {
        let dateFormatter = DateFormatter()
        dateFormatter.dateStyle = .short
        dateFormatter.timeStyle = .short
        return dateFormatter
    }()
}
]
```
So then you can use it as so:
``` swift
dateLabel.text = entry.timestamp.shortString
```